model/src/main/java/com/flipkart/flux/domain/ClientElb.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2012-2018, the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.flipkart.flux.domain;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+/**
+ * <code>ClientElb</code> represents Client Cluster's Elb details.
```

Comment:
Elaborate more in java doc, for eg Entity which contains info regarding the endpoint of a clients execution environment, maintained in mysql.

Reply:
Added more elaborative javadoc about ClientElb Entity.